### PR TITLE
chore(docs): Hide TOC in pages where only the title is displayed

### PR DESCRIPTION
**Pull request by davidblnc**
This PR aims to remove the table of contents in the docs where it's not necessary (example provided as screenshot).

imagen

### CODE REVIEW

**Comment by mcansh** [comment]
thanks! i’m wondering if it’s better to have this automated in our markdown parser. @ryanflorence

**Review by machour — CHANGES_REQUESTED** [review]
@davidblnc can you rebase this on top of main?
Seems like we do need a toc for browser support now, otherwise it looks ok for merge.

File: docs/guides/browser-support.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Browser Support
+toc: false
```
Comment (MichaelDeBoey) [inline docs/guides/browser-support.md:3]: Even though it's a small page, we have multiple headers on this page

```suggestion
```

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

**Comment by machour** [comment]
Superseded by #5249, thank you nonetheless @davidblnc  🙏🏼

**Pull request closed without merging**